Explorer/Assets/AddressableAssetsData/AssetGroups/Localization-Locales.asset

```diff
@@ -23,6 +23,18 @@ MonoBehaviour:
     m_SerializedLabels:
     - Locale
     FlaggedDuringContentUpdateRestriction: 0
+  - m_GUID: f81b53a5818cc4be9ab33b5208f77a18
```

Comment:
Are we again adding the Spanish and Italian? Or is that accidental?

Reply:
oh wow, totally missed this, I think it auto added them, let me remove

Reply:
Done